server/game/cards/09.1-FtE/PurityOfSpirit.js

```diff
@@ -0,0 +1,29 @@
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+const { CardTypes } = require('../../Constants');
+
+class PurityOfSpirit extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Choose a bushi character to honor',
+            condition: () => this.game.isDuringConflict(),
+            target: {
+                cardType: CardTypes.Character,
+                cardCondition: card => card.hasTrait('bushi') && card.isParticipating(),
+                gameAction: AbilityDsl.actions.multiple([
+                    AbilityDsl.actions.honor(),
+                    AbilityDsl.actions.delayedEffect({
+                        when : {
+                            onConflictFinished: () => true
+                        },
+                        gameAction: AbilityDsl.actions.discardStatusToken(context => ({ target: context.target.personalHonor })),
+                        message: 'remove the status token from {0}'
```

Comment:
This isn't going to show a very good message in chat.  You probably want something like:
```
message: '{2} removes the {3} from {1} due to the delayed effect of {0}',
messageArgs: context => [context.player, context.target.personalHonor]
```

Reply:
so it isn't possible to pass in the `messageArgs` to the function. the interface to `DelayedEffectActionProperties` only defines when, message and gameAction. so I assume this was the intent when the delay effect action was made. I did change the message to 
`'the status token has been removed from {1} due to the delayed effect of {0}'`
let me know if  this is a good message given that the message will only get an array with 'Purity of Spirit' and the name of the character effected